apps/remote_control/lib/lexical/remote_control/api.ex

```diff
@@ -47,6 +48,10 @@ defmodule Lexical.RemoteControl.Api do
     ])
   end
 
+  def struct_doc(%Project{} = project, struct_module) do
+    RemoteControl.call(project, Struct, :doc, [struct_module])
```

Comment:
this should go through `Completion`, and it should be called `documentation` not `doc`